### PR TITLE
Create some basic tests for kubernetes_e2e

### PR DESCRIPTION
**Pull request by krzyzacy**
at least prevent silly mistakes here

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -218,7 +214,7 @@ def sig_handler(self, _signo, _frame):
         check('docker', 'stop', self.container)
 
 
-def main(args):
+def run_e2e(args):
```
Comment (fejta) [inline scenarios/kubernetes_e2e.py]: Why change this from main?

File: scenarios/kubernetes_e2e.py [hunk 2]
```diff
@@ -295,50 +291,48 @@ def main(args):
...
         '--test', default='true', help='If we need to set --test in e2e.go')
-    PARSER.add_argument(
+    parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
-    ARGS = PARSER.parse_args()
+    return parser
 
-    main(ARGS)
+if __name__ == '__main__':
+    ARGS = create_parser().parse_args()
```
Comment (fejta) [inline scenarios/kubernetes_e2e.py]: Seems like we can eliminate the ARGS?

Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: done.

File: scenarios/kubernetes_e2e.py (hunk 1 above)
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: that's a good question.. let's keep it main

**Pull request merged**